libs/providers/flagd/src/lib/service/service.ts

```diff
@@ -7,11 +7,12 @@ export interface Service {
     disconnectCallback: () => void,
   ): Promise<void>;
   disconnect(): Promise<void>;
-  resolveBoolean(flagKey: string, context: EvaluationContext, logger: Logger): Promise<ResolutionDetails<boolean>>;
-  resolveString(flagKey: string, context: EvaluationContext, logger: Logger): Promise<ResolutionDetails<string>>;
-  resolveNumber(flagKey: string, context: EvaluationContext, logger: Logger): Promise<ResolutionDetails<number>>;
+  resolveBoolean(flagKey: string, defaultValue: boolean, context: EvaluationContext, logger: Logger): Promise<ResolutionDetails<boolean>>;
```

Comment:
Was the service interface exported publically? Should this be considered a breaking change?

Reply:
I can't import this locally I don't think it was every exported, so we should be good... cc @Kavindu-Dodan